pkg/test/resourcefixture/testdata/basic/secretmanager/v1beta1/fullsecretmanagersecret/_generated_object_fullsecretmanagersecret.golden.yaml

```diff
@@ -18,25 +17,23 @@ spec:
   annotations:
     bar: secretmanagersecret-bar
     foo: secretmanagersecret
-  expireTime: "2025-10-02T15:01:23Z"
+  expireTime: "2025-10-03T15:01:23Z"
```

Comment:
Hmmm . we should probably normalize this!

Reply:
Which format would you suggest to use ?  The current format is ISO 8601.